CoinChangerUnitTests/CoinChangerTest.cs

```diff
@@ -0,0 +1,110 @@
+using Xunit;
+using System.Collections;
+using System.Collections.Generic;
+
+
+namespace coinChanger
+{
+    public class TestClass
+    {
+        [Fact]
+        public void oneReturnsOnePenny()
+        {
+            MakeChange myCoinChanger = new MakeChange();
+            Assert.Equal(new int[] { 1 }, myCoinChanger.calculateChange(1));
+            Assert.Equal(new Dictionary<int, int> { { 1, 1 } }, myCoinChanger.countChange());            
+            Assert.Equal("1 has been converted to the following coins:\nx1 1\n", myCoinChanger.displayChange(1));
+        }
+
+        [Fact]
+        public void twoReturnsTwoPennies()
+        {
+            MakeChange myCoinChanger = new MakeChange();
+            Assert.Equal(new int[] { 1, 1 }, myCoinChanger.calculateChange(2));
+            Assert.Equal(new Dictionary<int, int> { { 1, 2 } }, myCoinChanger.countChange());
+            Assert.Equal("2 has been converted to the following coins:\nx2 1\n", myCoinChanger.displayChange(2));
+        }
+
+        [Fact]
+        public void fiveReturnsOneNickel()
+        {
+            MakeChange myCoinChanger = new MakeChange();
+            Assert.Equal(new int[] { 5 }, myCoinChanger.calculateChange(5));
+        }
+
+        [Fact]
+        public void sixReturnsOneNickelAndOnePenny()
+        {
+            MakeChange myCoinChanger = new MakeChange();
```

Comment:
I think it will make more sense to rename the MakeChange class:
```
CoinChanger      myCoinChanger             = new CoinChanger();
|_________|     |_____________|              |________________|
object type     instance variable name       new instance of the object

```

Reply:
ping me on slack and we can get on a call if this doesn't make sense